src/databaseSetup.js

```diff
@@ -1,5 +1,5 @@
 const Sequelize = require("sequelize");
-const { DB_NAME, DB_USER, DB_PASSWORD, DB_HOST, PORT } = require("./config");
+const { DB_NAME, DB_USER, DB_PASSWORD, DB_HOST, PORT } = require("./config.js");
```

Comment:
It doesn't require to add `.js` extension in `require` statement

Reply:
yep